app/models/article.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Article < ApplicationRecord
+  validates :message, :user, presence: true
+  validates :message, length: { in: 6...200 }
+
+  belongs_to :user
```

Comment:
https://api.rubyonrails.org/v7.0.3.1/classes/ActiveRecord/Associations/ClassMethods.html#method-i-belongs_to
```suggestion
  validates :message, presence: true
  validates :message, length: { in: 6...200 }

  belongs_to :user, optional: false
```

I think it is better to move this validation to the association definition.



or
```suggestion
  validates :message, presence: true
  validates :message, length: { in: 6...200 }

  belongs_to :user, validate: true
```